Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeAdPermissions.ps1

```diff
@@ -112,10 +113,15 @@ function Get-ExchangeAdPermissions {
 
             try {
                 try {
+                    # Check if AD split permissions is enabled and if so, throw to check for objectVersion instead ACE
+                    if (Get-ExchangeADSplitPermissionsEnabled) {
+                        throw "Active Directory split permissions enabled. Fallback to 'objectVersion (Default)' validation initiated."
```

Comment:
This approach to flow control should be reconsidered in the future, but I'm approving this for now.